### DISCUSSION TITLE
Using HTTPS Listener

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5074] [Q&A]
I need to convert a HTTP listener to a HTTPS listener.  Haven't really worked with this before but can I create my own cert to use (on the SSL Manager tab, lower section) and send that to the vendor connecting to us? After creating the cert, do I just export it for the vendor to use?

**Comment by pacmano1** [comment]
You can create your own certs but they are sooooo cheap these days I'd get a real one.  For a single endpoint as low as $4/year.  They can even be free with Lets Encrypt, but you would need to coordinate getting that cert to Mirth. For a nicer user interface for working with certificates try KeyStore Explorer. It's free. You would then import the JKS to Mirth.

**Comment by rutledj** [comment]
Understood about the cheapness but, for my needs, one I create myself will suffice. Just wasn't sure what needs to be given to the connecting party. Would I just use the Export button? If so, what options are needed?  

![image](https://user-images.githubusercontent.com/6484700/156626170-77567c77-09a0-4524-a7c9-d94a6179ff07.png)

**Reply by pacmano1** [reply]
Nothing needs to be given to the other party depending on what software they use.    They need to accept your fake cert.   

They could choose to add your fake certificate authority, but in general that is not a good practice.
